cmd/cmd_pkg.go

```diff
@@ -219,9 +223,53 @@ func loadCommandHandler(cmd *cobra.Command, args []string) {
 	buildImageFlags := NewBuildImageCommandFlags(flags)
 	runLocalInstanceFlags := NewRunLocalInstanceCommandFlags(flags)
 	pkgFlags := NewPkgCommandFlags(flags)
-
 	pkgFlags.Package = args[0]
 
+	for _, port := range runLocalInstanceFlags.Ports {
+		conn, err := net.DialTimeout("tcp", ":"+port, time.Second)
+		if err != nil {
+			continue // assume port is not being used
+		}
+
+		if conn != nil {
+			conn.Close()
+
+			var (
+				pid     string
+				message = fmt.Sprintf("Port %v is being used by other application", port)
+			)
+
+			if runtime.GOOS == "linux" || runtime.GOOS == "darwin" {
```

Comment:
I'd prefer to keep architecture-specific code in separated files like `lepton/ldd_linux.go` or `lepton/ldd_darwin`.

Reply:
Fixed by https://github.com/nanovms/ops/pull/1025/commits/991df4ea689ffaeab24358c7c00c80f62c20f7b2